GVRf/Extensions/gvrf-particlesystem/src/main/java/org/gearvrf/particlesystem/GVRPlaneEmitter.java

```diff
@@ -0,0 +1,191 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gearvrf.particlesystem;
+
+/**
+ * Is a GVREmitter of the plane shape. The particles are emitted from
+ * a planar area in the X-Z plane of a certain width and height in a positive y-direction with a
+ * random position and velocity. The direction of emmission can be changed by
+ * rotating the emitter in the application.
+ */
+
+import org.gearvrf.GVRContext;
+import org.gearvrf.GVRDrawFrameListener;
+import org.gearvrf.scene_objects.GVRTextViewSceneObject;
+import org.gearvrf.utility.Log;
+import org.joml.Vector3f;
+
+import java.lang.ref.WeakReference;
+import java.util.Random;
+
```

Comment:
I think the javadoc should go right in front of the class to be picked up by the generator.